src/main/java/ch/epfl/biop/imaris/command/MakeSurfaceCommand.java

```diff
@@ -30,16 +30,16 @@
     @Override
     public void run() {
         try {
-            ISurfacesPrx surf = SurfacesDetector.Channel(channelIndex)
+            ISurfacesPrx surf = EasyXT.Surfaces.create(channelIndex)
                     .setSmoothingWidth(smoothingWidth)
                     .setLowerThreshold(lowerThreshold)
                     .setName(surfaceName)
-                    .setColor(new Integer[]{color.getRed(),color.getGreen(),color.getBlue()})
+                    .setColor(new Integer[]{color.getRed(), color.getGreen(), color.getBlue()})
                     .build()
                     .detect();
 
             // Adds the surface to the scene
-            EasyXT.getImaris().GetSurpassScene().AddChild(surf,0);
+            EasyXT.Scene.addToScene(surf);
```

Comment:
following previous comment  "EasyXT.Scene.addToScene(surf)" could be changed to EasyXT.Scene.add(surf)

Reply:
changed to `Scene.putItem()`